app/utils.js

```diff
@@ -169,3 +178,218 @@ exports.signXrpWithPrivateKey = function(txHex, privateKey, options) {
     id: exports.computeBinaryTransactionHash(serialized)
   };
 };
+
+
+exports.UserInput = function(args) {
```

Comment:
All these UserInput functions were copied over from bitgo-cli

Reply:
Is this still urgent, or can we spend some time modernizing? If possible let's use class syntax